cocos/2d/CCLabel.h

```diff
@@ -749,9 +762,8 @@ class CC_DLL Label : public Node, public LabelProtocol, public BlendProtocol
     Color4F _textColorF;
 
     QuadCommand _quadCommand;
-    CustomCommand _customCommand;
-    CustomCommand _customCommandOutLine;
-    CustomCommand _customCommandShadow;
+
+    std::vector<BatchCommand> _batchCommands;
```

Comment:
Why need vector?

Reply:
Each `_batchNodes` need a group of different renderer commands.